spec/features/applications/new_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_helper"
+
+RSpec.describe "As a visitor" do
+  # 2. Application new Page
+  describe "When I visit the pet index page" do
+    it "Then I see a link to 'Start an Application'" do
+      shelter = Shelter.create(name: "Aurora shelter", city: "Aurora, CO", foster_program: false, rank: 9)
+      pet_1 = Pet.create(adoptable: true, age: 1, breed: "sphynx", name: "Lucille Bald", shelter_id: shelter.id)
+
+      visit "/pets"
+
+      click_link("Start an Application")
+      expect(page).to have_current_path("/applications/new") 
+    end
```

Comment:
I would probably also put this test inside of your `spec/pets/index_spec.rb` because we expect there to be a link on your pet index page as well.